src/public/resolvers/queries/CorpusItem.ts

```diff
@@ -19,3 +22,19 @@ export async function getCorpusItem(item, { db }): Promise<CorpusItem> {
 
   return getCorpusItemFromApprovedItem(approvedItem);
 }
+
+export async function getSavedCorpusItem(
+  item,
+  args,
+  { db }
+): Promise<CorpusItem> {
+  const { url } = item;
+
+  const approvedItem = await getApprovedItemByUrl(db, url);
+  if (!approvedItem) {
+    console.log(`Could not find Corpus Item with Url of "${url}"`);
```

Comment:
this is a huge nitpick for this PR, but - given the volume of attempts at connecting a save to a corpusitem, i am afraid this `console.log` is going to flood our logs, making it difficult to find anything else. (i just ran into this with the syndication api migration.)

i'm generally in favor of a good `console.log`, but i'm not sure what purpose it's serving here.

please ignore me if the volume here will be far lower than i'm guessing.

Reply:
Yea that's fair we copied the pattern from syndication where it does the same thing.